extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -651,10 +653,7 @@ private function doSearchTags( $searchTag ) {
 	/**
 	 * doMultiRemoveTags
 	 *
-	 * @access private
-	 * @author uberfuzzy@wikia-inc.com
-	 *
-	 * @return text message (use Wikia::*box functions)
+	 * @return string Text message (use Wikia::*box functions)
 	 */
 	private function doMultiRemoveTags( ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)